data/builtins/thingengine.builtin/manifest.tt.in

```diff
@@ -196,9 +196,23 @@ class @org.thingpedia.builtin.thingengine.builtin
   #[confirm=false];
 
   action alert()
-  #_[result="Time's up!"]
+  #_[result="Alert"]
+  #_[formatted=[
+    { type="sound", name="alarm-clock-elapsed", exclusive=true }
+  ]]
   #_[canonical="alert"]
-  #[doc="makes Genie show/say a generic alert"]
+  #[doc="makes Genie show/play a generic alert"]
+  #[confirm=false];
+
+  action timer_expire(out message_output: String
```

Comment:
It's usually better to return structured information from Thingpedia functions, and use the result or formatted phrases to put them into words. The reason is postprocessing and locale-specific formatting, which otherwise you need to duplicate into your code.